docs/content/basic/authenticating-oauth.md

```diff
@@ -4,256 +4,703 @@ lang: en
-  installerOptions: {
-      authVersion: 'v1', // default  is 'v2', 'v1' is used for classic slack apps
-      metadata: 'some session data',
-      installPath: '/slack/installApp',
-      redirectUriPath: '/slack/redirect',
-      userScopes: ['chat:write'],
-      callbackOptions: {
-        success: (installation, installOptions, req, res) => {
-          // Do custom success logic here
-          res.send('successful!');
-        }, 
-        failure: (error, installOptions , req, res) => {
-          // Do custom failure logic here
-          res.send('failure');
-        }
-      },
-      stateStore: {
-        // Do not need to provide a `stateSecret` when passing in a stateStore
-        // generateStateParam's first argument is the entire InstallUrlOptions object which was passed into generateInstallUrl method
-        // the second argument is a date object
-        // the method is expected to return a string representing the state
-        generateStateParam: async (installUrlOptions, date) => {
-          // generate a random string to use as state in the URL
-          const randomState = randomStringGenerator();
-          // save installOptions to cache/db
-          await myDB.set(randomState, installUrlOptions);
-          // return a state string that references saved options in DB
-          return randomState;
-        },
-        // verifyStateParam's first argument is a date object and the second argument is a string representing the state
-        // verifyStateParam is expected to return an object representing installUrlOptions
-        verifyStateParam:  async (date, state) => {
-          // fetch saved installOptions from DB using state reference
-          const installUrlOptions = await myDB.get(randomState);
-          return installUrlOptions;
-        }
-      },
-  }
-});
+const database = {
+  store: {},
+  async get(key) {
+    return this.store[key];
+  },
+  async delete(key) {
+    delete this.store[key];
+  },
+  async set(key, value) {
+    this.store[key] = value;
+  },
+};
 ```
-</details>
\ No newline at end of file
+
+</details>
+
+### Sign in with Slack
+
+Right now Bolt does not support [Sign in with Slack][siws] out-of-the-box. This
+still continues to remain an option using APIs from the
+[`@slack/web-api`][web-api] package, which aim to make implementing OpenID
+Connect (OIDC) connections simple. Alternative [routes][custom-routes] might be
+required.
+
+Explore [this relevant package documentation][oidc] for reference and example.
+
+### Extra authorizations
+
+If you need additional authorizations, such as user tokens from users of a team
+where your app is already installed, or have a reason to dynamically generate an
+install URL, an additional installation is required.
+
+Generating a new installation URL requires a few steps: manually instantiate an
+`ExpressReceiver`, assign the instance to a variable named `receiver`, and then
+call `receiver.installer.generateInstallUrl()`.
```

Comment:
```suggestion
Generating a new installation URL requires a few steps: 
1. Manually instantiate an `ExpressReceiver` instance.
2. Assign the instance to a variable named `receiver`.
3. Call the `receiver.installer.generateInstallUrl()` function.
```

Reply:
Didn't realize why listing these steps inline felt so wrong! 🤪  Thanks for this suggetsion 🙏